### PR TITLE
Added environment variable not to run config:cache

### LINKED ISSUE #105: Option to not run config:cache?
Could there be an option to not run config:cache?

We are a development locally using `bref/php-*-fpm-dev:2`.
Every time we change the `.env`, we have to restart the docker container, which is very tedious.

### PR DESCRIPTION
**Pull request by sunaoka**
Addition of environment variable `BREF_LARAVEL_CACHE_CONFIG` that does not run config:cache.

Fixed #105

I didn't know how to write the test, so I checked it manually.

# Local development (Docker Compose)

```yaml
services:
  app:
    environment:
      BREF_LARAVEL_CACHE_CONFIG: 0 # true, false, 0, 1
```

# Lambda

```yaml
functions:
  app:
    environment:
      BREF_LARAVEL_CACHE_CONFIG: 0 # true, false, 0, 1
```

### CODE REVIEW

File: src/bref-init.php [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
     MaintenanceMode::setUp();
 
+    $shouldCache = env('BREF_LARAVEL_CACHE_CONFIG', env('APP_ENV') !== 'local');
+
+    if (! $shouldCache) {
+        return;
+    }
```
Comment (mnapoli) [inline src/bref-init.php:22]: The `if` is too early: it will break other things (like XDG config)

Comment (sunaoka) [reply src/bref-init.php:22]: Thanks for the review.
I will move it after setting XDG_CONFIG_HOME.

Comment (sunaoka) [reply src/bref-init.php:27]: Changed `config:cache` to determine if it should run after setting `XDG_CONFIG_HOME`.

**Review by mnapoli — APPROVED** [review]
LGTM, I'll let @tillkruss or @georgeboot review, and then I'll merge and update the bref.sh docs.

**Pull request merged**